content/docs/getting-started/prerequisites.md

```diff
@@ -15,7 +15,7 @@ To follow the steps described in this section, you will need to:
 
 ## Install the OCM Command Line Interface (CLI)
 
-The CLI is used to interact with component versions and registries. Install it like described in [Installing the OCM CLI](/docs/getting-started/installing-the-ocm-cli).
+The CLI is used to interact with component versions and registries. Install it like described in [Installing the OCM CLI](https://ocm.software/docs/getting-started/installing-the-ocm-cli).
```

Comment:
why do we need absolute links now? was it broken before?

Reply:
the way how Hugo handles relative links is not compatible with standard HTML link checkers, neither it is with markdown/HTML link checks in IDEs. They are always shown as broken. That is why I replaced all relative links with absolute ones. `This is a workaround`and something I will pick up for the next version of the website, as the target should for sure be to keep relative links wherever possible. If we stay with Hugo we would introduce some custom code to check relative links.

Reply:
since there's still the option for having a different web server, e.g. Docusaurus, I tend to not don't invest anything in this relative link issue now, as it might turn out that what we invest in, will not be used in the future.

Reply:
I think this option is off the table for now. We should double down on Hugo and roll with it.

Reply:
>> I think this option is off the table for now. 
yes, but this does not mean that we will re-introduce the relative links that we're not able to check on in this PR. Right now quality matters most and I don't want to have any broken link on the page and right now there ist no other way of introducing absolute links . I added a new sub-task in https://github.com/open-component-model/ocm-project/issues/431 that will take care that we will have script being able to test relative links in the Hugo notation.